server/main.lua

```diff
@@ -162,7 +162,7 @@ lib.addCommand('kill', {
 		exports.qbx_core:Notify(source, Lang:t('error.not_online'), 'error')
 		return
 	end
-	lib.callback('qbx_medical:client:killPlayer', args.id)
+	lib.callback.await('qbx_medical:client:killPlayer', args.id)
```

Comment:
Ditto

Reply:
This one could be a net event, but also works fine as a callback.